.github/issue_template.md

```diff
@@ -0,0 +1,17 @@
+**Important:** GitHub issues are for feature requests or bug reports. The Capistrano team recommends you use [Stack Overflow](http://stackoverflow.com/questions/tagged/capistrano) for general questions. For more details, please see our [contribution policy](https://github.com/capistrano/capistrano/blob/master/CONTRIBUTING.md).
+
+---
+
+### Steps to reproduce
+If at all possible, please try to create a new Capistrano example project which reproduces the issue you are seeing. You can then upload the individual files to a GitHub Gist or a GitHub project. Often times, an issue is resolved simply by making this test case.
```

Comment:
I agree this makes the maintainers' lives easier, but I think this is daunting to ask of someone reporting a bug. You can't really create a  working example Capistrano project without real servers and a `:repo_url` that points to real source code, none of which can be accessed without the reporter's credentials. Unless I am misunderstanding what you are asking for here?